ios/ExposureProcessor.swift

```diff
@@ -90,8 +90,12 @@ public class ExposureProcessor {
             result["type"] = ["bluetooth"]
         case .restricted:
             result["state"] = "restricted"
+        case .paused:
+            result["state"] = "disabled"
+            result["type"] = "paused"
         default:
             result["state"] = "unavailable"
+            result["type"] = "initialising"
```

Comment:
We should add these new values to the README (see https://github.com/covidgreen/react-native-exposure-notification-service#status)

Reply:
Readme updated